src/entity/release.rs

```diff
@@ -253,6 +255,7 @@ pub struct Track {
     pub length: Option<u32>,
     pub position: u32,
     pub id: String,
+    pub artist_credits: Option<Vec<ArtistCredit>>
```

Comment:
Shouldn't it be singular?

```suggestion
    pub artist_credit: Option<Vec<ArtistCredit>>
```

Reply:
I haven't tested it yet as it's only a draft, but yeah.